src/EditorFeatures/Core.Wpf/InlineParameterNameHints/InlineParameterNameHintsTag.cs

```diff
@@ -18,18 +33,93 @@ namespace Microsoft.CodeAnalysis.Editor.InlineParameterNameHints
     internal class InlineParameterNameHintsTag : IntraTextAdornmentTag
     {
         public const string TagId = "inline parameter name hints";
+        private readonly IToolTipService _toolTipService;
+        private readonly ITextView _textView;
+        private readonly SnapshotSpan _span;
+        private readonly SymbolKey _key;
+        private readonly IThreadingContext _threadingContext;
+        private readonly Lazy<IStreamingFindUsagesPresenter> StreamingPresenter;
```

Comment:
```suggestion
        private readonly Lazy<IStreamingFindUsagesPresenter> _streamingPresenter;
```

If it's private, we use the _ naming convention.